### ISSUE TITLE
[d3d9] Mercenaries 2 World in flame: game crashes after in ~5 seconds of gameplay.

### ISSUE THREAD

**Issue by CortexReaver** [doitsujin/dxvk#3468]
It's working smooth without any wrapper on Win10, but decided to report it anyway.

### System information
- GPU: GTX 1050 Ti
- Driver: 531.79 (and early)
- OS: Windows 10 2H22
- DXVK version: 2.1, 2.2
(Note: It seems that v2.0 does not crash, but stutters as hell )

### Apitrace file(s)
Can't apitrace because of this error: 
![01-06-2023 045926](https://github.com/doitsujin/dxvk/assets/34350690/dd0a00cb-428f-4fc3-8ab4-e0bd7eabf217)

### Log files
2.2 logs: 
[Mercenaries2_d3d9.log](https://github.com/doitsujin/dxvk/files/11620188/Mercenaries2_d3d9.log)
[d3d.log](https://github.com/doitsujin/dxvk/files/11620189/d3d.log)

2.0 logs(does not crash but stutters):
[Mercenaries2_d3d9.log](https://github.com/doitsujin/dxvk/files/11620196/Mercenaries2_d3d9.log)
[d3d.log](https://github.com/doitsujin/dxvk/files/11620197/d3d.log)

**Comment by CortexReaver** [comment]
> Haven't looked at this yet but note that the above config doesn't do anything with this dxvk
That's because I've tested it with differenr versions, including "async" one. Or whatever it called, the entirely different github page. Config is just a "leftovers".

**Comment by Blisto91** [comment]
I did some testing on Linux with AMD and radv but didn't encounter any crashing issues. Gonna try Windows later

**Comment by CortexReaver** [comment]
Works fine *fingers crossed* with this build: https://github.com/doitsujin/dxvk/actions/runs/5372617181 But there is still some very annoying "bug": if you alt-tab too much, game stops capturing by OBS and the picture freezes in OBS preview. Other than this everything seems fine.

**Comment by CortexReaver** [comment]
Game was crashing after cutscenes (some time after you exit quest buildings). if you need it, I can record an apitrace later.

**Comment by Blisto91** [comment]
Yes please.

**Comment by CortexReaver** [comment]
> ### Apitrace file(s)
> Can't apitrace because of this error: ![01-06-2023 045926](https://user-images.githubusercontent.com/34350690/242457492-dd0a00cb-428f-4fc3-8ab4-e0bd7eabf217.jpg)

Oh, yes, I forgot about this error. So no apitrace, sorry :( Only can record a video of a glitch.

**Comment by Blisto91** [comment]
Can you also share a save game if it's in a specific location? :slightly_smiling_face:

**Comment by Blisto91** [comment]
@CortexReaver Friendly ping

**Comment by CortexReaver** [comment]
> @CortexReaver Friendly ping

Yeah, I am so sorry. I've been really busy lately. I'll provide all necessary files when I can.

**Comment by Blisto91** [comment]
@CortexReaver Might be interesting to give it another shot with current master when you have time
https://github.com/doitsujin/dxvk/actions/runs/7797876671

**Comment by Blisto91** [comment]
@CortexReaver Friendly ping. Could you retry with 2.3.1
